### DISCUSSION TITLE
iNav angle mode lean to left ahead with a certain speed (NOT drift!!!)

### DISCUSSION THREAD

**Discussion by lida2003** [iNavFlight/inav#8625] [Q&A]
Well, I calibrated quard with IMU & optical flow & lidar sensors.

It seems that iNav angle mode will lean to left ahead with quite a fast speed, NOT dirft. 

Is there any way to configure iNav much more horizontal, BF seems more effective when IMU configured.

![图片](https://user-images.githubusercontent.com/1296171/206968321-67f7ba9b-c3a8-4ce7-bf90-3ce941a992a5.png)

![图片](https://user-images.githubusercontent.com/1296171/206975331-ad6d3a56-31ae-4862-ad9b-c0b7096e5f01.png)

**Comment by lida2003** [comment]
suggest way like BF do， which can simply configure the IMU

![图片](https://user-images.githubusercontent.com/1296171/206975446-fa11e6b1-9063-449b-8892-0c445127b555.png)

**Reply by MrD-RC** [reply]
One of the IMU calibration steps in INAV is level. The other 5 steps improve the calibration, by taking all axis in to account.

Your craft rolling isn't necessarily the board alignment. Have you accidentally added trim on the transmitter? Does the roll input centre at 1500 on the receiver page? If they are both OK, you can trim the board alignment quite easily in the CLI.

**Reply by lida2003** [reply]
One of the IMU calibration steps in INAV is level. The other 5 steps improve the calibration, by taking all axis in to account.

==> OK

Your craft rolling isn't necessarily the board alignment. 

==> I didn't get your point, what do u mean?

Have you accidentally added trim on the transmitter?

==> RC remote control trim is centered but the craft will move left ahead continuely.
 The above Fig. is what I have adjust trims for compansation to make the craft take off at the same spot (maybe some drift, but NOT move left ahead) 

Does the roll input centre at 1500 on the receiver page? 

==> I think the stick is ouput 1500, as my 5 inch quad works fine with the same RC remote controller.

If they are both OK, you can trim the board alignment quite easily in the CLI.

==> Any link avaliable? I can try this if vital adjustment can be done by CLI. I'm from betaFlight, so NOT that familiar with iNav calibration.
